app/src/org/commcare/activities/components/FormEntryDialogs.java

```diff
@@ -31,7 +31,7 @@ public static void createQuitDialog(final FormEntryActivity activity, boolean is
         View.OnClickListener stayInFormListener = v -> activity.dismissAlertDialog();
         DialogChoiceItem stayInFormItem = new DialogChoiceItem(
                 StringUtils.getStringRobust(activity, R.string.do_not_exit),
-                R.drawable.ic_blue_forward,
+                LocalePreferences.isLocaleRTL() ? R.drawable.ic_blue_backward : R.drawable.ic_blue_forward,
```

Comment:
It seems like at some places we are switching the icons (like `icon_chevron_right_attnpos`) and at others we are calling `mirrorView` on Views. Curious whats the reasoning behind choosing one method over another. I would prefer if we can achieve mirroring without adding any new icons.

Reply:
👍